src/hb-serialize.hh

```diff
@@ -277,11 +279,47 @@ struct hb_serialize_context_t
 
     auto& link = *current->links.push ();
     link.is_wide = sizeof (T) == 4;
+    link.is_signed = false;
+    link.is_absolute = false;
     link.position = (const char *) &ofs - current->head;
     link.bias = (const char *) base - current->head;
     link.objidx = objidx;
   }
 
+  /*
+   * Set a link to an object with an absolute offset.
```

Comment:
Can you clarify in this comment what an absolute offset is. My understanding is this means from the start of the serialize context.